File: 프로그래머스/0/181949. 대소문자 바꿔서 출력하기/대소문자 바꿔서 출력하기.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        String result = "";
        for (int i = 0; i < str.length(); i++) {
            char c = str.charAt(i);
            if (Character.isUpperCase(c)) {
                result += Character.toLowerCase(c);  // 대문자 → 소문자 [web:52]
            } else {
                result += Character.toUpperCase(c);  // 소문자 → 대문자 [web:52]
            }
        }
        System.out.println(result);
    }
}

File: 프로그래머스/0/181949. 대소문자 바꿔서 출력하기/대소문자 바꿔서 출력하기.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.on('line', function (line) {
    let result = '';
    for (let char of line) {
        if (char === char.toUpperCase()) {
            result += char.toLowerCase();  // toggle case [web:56]
        } else {
            result += char.toUpperCase();
        }
    }
    console.log(result);
    rl.close();
});
